docs/getting-started/c_best_practices.md

```diff
@@ -14,7 +14,7 @@ If your requests are spending a lot of time in the queue, it is possible that ot
 
 If you find that another user has submitted a lot of requests \(especially complex requests that take a long time to process\), it is usually a good idea to check with the requestor to make sure that the requests are valid. You can cancel waiting requests by using Report.Cancel.
 
-Note that The API system is throttled on a company level, mainly to prevent DDoS type of attacks or buggy scripts from unnecessarily taxing the system. The limit is high enough that 99.9% of the our customers will never hit the limit. If you hit this limit, you will receive a Request.OverLimit error message, and you can contact Adobe to have your throttle level increased.
+Note that The API system is throttled on a company level, mainly to prevent DDoS type of attacks or buggy scripts from unnecessarily taxing the system. Both the gate and load balancer limit is 2 MB. If you hit this limit, you will receive a Request.OverLimit error message, and you can contact Adobe to have your throttle level increased.
```

Comment:
This seems to suggest that rate limits and request size limits the same thing. These two limits are enforced by different mechanisms. The `Request.OverLimit` response is only effective for rate limits and not request size limits. I am not sure what the responses are for too large of requests

Reply:
Great catch. Thanks. I will change.